package.json

```diff
@@ -1,12 +1,26 @@
 {
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "prettier --no-semi --single-quote --write",
```

Comment:
You can configure Prettier via `prettier` key in the `package.json`, so let's move prettier config there to reduce duplication (see [line 15](https://github.com/andywer/webpack-blocks/pull/263/commits/f2dc22756be23685e767f20196e79aaeb8fb9c26#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15))